### ISSUE TITLE
Remove prompt without affecting a meeting

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#4238]
There are currently 2 problems with removing retro template prompts:

1. Removing a prompt that is currently being used in a meeting causes the meeting to crash
2. Removing a prompt removes that prompt from all completed meetings, even if it was used in that meeting

### AC

- [ ] Remove a prompt that is being used in a meeting without the meeting being effected, i.e. it doesn't crash and the prompt doesn't get removed
- [ ] Remove a template prompt that was used in a completed meeting without removing the prompt from the completed meeting

**Comment by ackernaut** [comment]
So what’s the plan here? Could we automatically clone and archive the template being used by the meeting while the original template is the one that’s being modified in the editor?

**Comment by nickoferrall** [comment]
I think that would work. After getting the green light from Matt, I implemented a solution in #4236 that resolves issue 1 by using `safeRemoveNodeFromArray` and resolves issue 2 by adding a `removedAt` field to the `TemplatePrompt`. We can then only show prompts if:

1. The prompt was created before the meeting was created
2. The prompt has not been removed or, if it has been removed, it was removed after the meeting was created

There's a console error I'd like to squash but once that's done, I think it should fix these issues.
